### PR TITLE
Add Browserbase to members.csv

### PR DESCRIPTION
**Pull request by pkiv**
Hi everyone! 

We're excited to join the OSS Pledge. 

* Blog Post: https://www.browserbase.com/blog/browserbase-pledge-to-open-source
* Brand Materials: Attached to this PR!

Bb-logo-june-3-2024-darkbg
Bb-logo-june-3-2024-lightbg
Bb-logo-june-3-2024-nobg
Bb-logo-june-3-2024-nobg
Bb-logomark-june-3-2024-nobg

### CODE REVIEW

**Review by vladh — APPROVED** [review]

**Comment by vladh** [comment]
@pkiv Great to have you on board! 🎉  I love the blog post.

Everything LGTM, but I'll wait for a second review from @chadwhitacre.

**Comment by chadwhitacre** [comment]
Woo-hoo! Welcome aboard! :D

**Pull request merged**
